src/manage_nsfs/health.js

```diff
@@ -343,34 +360,34 @@ class NSFSHealth {
         return account_details;
     }
 
+    async get_connection_status() {
+        const connection_details = await this.get_storage_status(TYPES.CONNECTION, this.all_connection_details);
+        return connection_details;
+    }
+
     async get_storage_status(type, all_details) {
         const invalid_storages = [];
```

Comment:
do we still need this two consts?

Reply:
Two consts?
There's "const connection_details", which generally looks good to me.
This method (get_connection_status) follows the pattern of the other get_bucket/account_status functions.